Assets/Scripts/Transition/InteractableObject.cs

```diff
@@ -19,25 +19,20 @@ void Awake()
         transitioned?.OnSpring.AddListener(seasonChecker);
     }
 
-    // Update is called once per frame
-
-
     public void seasonChecker()
     {
-        if (colliderOnSpring)
+        if (colliderOnSpring && transitioned.IsSpring)
```

Comment:
That's how I did it at the start but the true and false I was getting wasn't right

Reply:
@atheer10 
Yeah I made a mistake in my script.
I was invoking the event before setting the boolean values.